addons/l10n_eu_service/tests/test_oss.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+
+from odoo.addons.l10n_eu_service.models.eu_tag_map import EU_TAG_MAP
+from odoo.addons.account.tests.account_test_savepoint import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install_l10n', '-at_install')
+class TestOSSBelgium(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(self, chart_template_ref='l10n_be.l10nbe_chart_template'):
+        try:
+            super().setUpClass(chart_template_ref=chart_template_ref)
+        except ValueError as e:
+            if e.args[0] == "External ID not found in the system: l10n_be.l10nbe_chart_template":
+                self.skipTest(self, reason="Belgian CoA is required for this testSuite but l10n_be isn't installed")
+            else:
+                raise e
+        self.company_data['company'].country_id = self.env.ref('base.be')
+        self.company_data['company']._map_eu_taxes()
+
+    def test_country_tag_from_belgium(self):
+        # get an eu country which isn't the current one:
+        another_eu_country_code = (self.env.ref('base.europe').country_ids - self.company_data['company'].country_id)[0].code
+        tax_oss = self.env['account.tax'].search([('name', 'ilike', f'%{another_eu_country_code}%')], limit=1)
+
+        for doc_type, report_line_xml_id in (
+                ("invoice", "l10n_be.tax_report_line_47"),
+                ("refund", "l10n_be.tax_report_line_49"),
+        ):
+            with self.subTest(doc_type=doc_type, report_line_xml_id=report_line_xml_id):
+                oss_tag_id = getattr(tax_oss, f"{doc_type}_repartition_line_ids")\
```

Comment:
you should never use `getattr` to access a field value by name, and you don't need it

```suggestion
                oss_tag_id = tax_oss[f"{doc_type}_repartition_line_ids"]\
```

Reply:
And the missing doc reference... ;-)
https://www.odoo.com/documentation/15.0/developer/reference/backend/security.html#accessing-object-attributes